components/Layout/Header/index.js

```diff
@@ -1,8 +1,20 @@
 import styled from "styled-components";
+import { Croissant_One } from "@next/font/google";
+
+export const croissantOne = Croissant_One({
+  subsets: ["latin"],
+  weight: "400",
+});
 
 const StyledHeader = styled.h1`
   text-align: center;
-  margin: 1rem;
+  margin: 0.1rem;
+  border-radius: 1rem;
+  background-color: var(--dark-green);
+  color: var(--gold);
+  padding: 0.5rem;
+  font-family: ${croissantOne.style.fontFamily};
```

Comment:
_Just a tiny suggestion. 🤏_

**Font Fallback:**
If the font from Croissant_One doesn't load or is not found, your component might not display text correctly. Consider adding a fallback font.

```suggestion
  font-family: ${croissantOne.style.fontFamily}, sans-serif;
```